internal/controller/server/execution/execution_test.go

```diff
@@ -114,13 +115,13 @@ func TestUpdateExecutorStatus(t *testing.T) {
 	}
 	executorRepoMock.On("Get", "exec 1").Return(&executorCPproto.ExecutorInfo{}, nil)
 	executorRepoMock.On("UpdateStatus", "exec 1", "free").Return(nil)
-	res, err := testExec.UpdateExecutorStatus(ctx, &request)
+	res, err := testExec.UpdateExecutorStatus(ctx, &request, 20*time.Second)
 	_, ok := getActiveExecutorMap(testExec.(*execution)).Get("exec 1")
 	assert.Equal(t, res.Recieved, true)
 	assert.Nil(t, err)
 	assert.Equal(t, true, ok)
-
-	res, err = testExec.UpdateExecutorStatus(ctx, &request)
+	pingTimeOut := 20 * time.Second
```

Comment:
can we move this 20 to constants, looking like a magic variable, and is being used at plenty of places.

Reply:
this is a test file variable. I am not sure if we should add a constant here. Also this is mostly testbased stuff